src/docs/New.mdx

```diff
@@ -0,0 +1,51 @@
+---
+name: New
+route: /new
+---
+
+import { Playground } from 'docz'
+import New from '../New'
+
+# New
+
+The New component provides a way to render elements, to create a resource and show or use the response data.
+
+## API
+
+| Property      | Type               | Required | Default    | Description                                                                                          |
+| ------------- | ------------------ | -------- | ---------- | ---------------------------------------------------------------------------------------------------- |
+| name          | String             | true     | -          | It must be equal to the reducer name. Defines the deep properties used in the component. Ex: foo.bar |
+| render        | Func               | true     | -          | A function returning a React Node. Ex: ({ create, creating = bool, error }, props) -> < JSX />       |
+| renderCreated | Func               | false    | -          | A function returning a React Node. Ex: (created = {}) -> < JSX />                                    |
+| parentId      | String/Number      | false    | -          | Parent element's Id, for more specificity in changes.                                                |
+| path          | String             | false    | `/${name}` | Ex: /foo/bar-\_12?&=                                                                                 |
+| method        | Enum (post or put) | false    | `post`     | The http request method.                                                                             |
+
+## How to use
+
+You have to add a reducer with the same name that you will pass to New.
+
+```javascript
+import { createReducer } from 'croods'
+
+const reducers = {
+  foobar: createReducer('foobar'),
+}
+```
+
+### render
+
+The render function receives three important parameters, **create** (func), **creating** (bool) and **error** (string).
+
+- The **create** function makes a `POST` request on `path` with the jsonified object parameter as body.
+- The **creating** is a bool to identify if the request is being done.
+- The **error** is populated after the use of create if the server response has an error status code.
+
+### renderCreated
+
+The renderCreated function is fired when the create function is used and the server response has a 2xx status.
```

Comment:
```suggestion
The renderCreated function is fired when the `create` function is called and the server response if of a 2xx status.
```

Reply:
`has` per `if of` is right?

Reply:
is =D

Reply:
I meant... it was supposed to be `is` not `it` lol